cnv/cnv_users_guide/cnv-ansible-modules.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// *
```

Comment:
And I can see that this module isn't added to an assembly yet (should it be now?), but when you're ready, then it should be updated here to list which assembly the module is a part of.

Reply:
@bergerhoffer One of us from CNV docs is going to do a follow-up PR to create an assembly for this module.